### ISSUE TITLE
No active keyboard focus is present on left pane tree elements of workbook window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#376]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF platform. Navigate to Help-> Tutorial -> Hello Workbooks and invoke it.
3. Navigate to "Hello Workbooks" tree element on left side pane.
### Expected Behavior
Active keyboard should be present on "Hello Workbooks" tree element in left pane of workbooks.
### Actual Behavior
Active keyboard focus is not present "Hello Workbooks" tree element present in left pane of workbooks.
Inspect Behavior:
Through the inspect, focus can be seen on left pane tree element "Hello Workbooks". Please refer below screenshot.
![screenshot 177](https://user-images.githubusercontent.com/22956130/40763808-dae54326-64c3-11e8-8571-71089f75e915.png)

### User Impact:
User will not know whether the element in left pane got the focus or not.
### Screenshot:
![screenshot 178](https://user-images.githubusercontent.com/22956130/40763819-e1026c84-64c3-11e8-9ac0-c859de85ee20.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.1.1

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.1.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
